package/yast2-bootloader.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Nov  1 11:49:05 UTC 2016 - jreidinger@suse.com
+
+- start using system locale when generating grub menu (bsc#998152)
```

Comment:
When I read "Using the system locale" I understand that it is using `LANG` (as opposed to, say always English). But in fact the fix makes it use `RC_LANG`. So I'd make it clear with

> Use the system-wide locale (/etc/sysconfig/language:RC_LANG) when generating the GRUB menu (bsc#998152)